R/dvhydrograph-render.R

```diff
@@ -398,8 +398,8 @@ plotDVHydroLegend <- function(plot_object, startDate, endDate, timezone, initial
   legend_offset <- ifelse(ncol==2, initialOffset+(0.025*leg_lines), initialOffset/2+(0.025*leg_lines))
   legend_offset <- legend_offset * modOffset
 
-  #If the time period is greater than 1 year additional x-axis labels are added so we must move the legend down further
-  legend_offset <- ifelse(as.period(interval(startDate, endDate, tzone = attr(startDate, timezone))) < lubridate::years(1), legend_offset+0.03, legend_offset+0.08)
+  #Add space to legend so that it doesn't overlap with dates.
+  legend_offset <- legend_offset + 0.8
```

Comment:
Number of legend items shouldn't make a difference - the offset defines the top of the legend and then everything else is pushed under it. 

We just discussed this - if the > 1 year plots look fine w/ 0.08, then this fix is good. If not, @dpattermann-usgs will change the offset values and clean up that if-else so it's not so giant